example/server.js

```diff
@@ -140,6 +140,22 @@ function init (config, callback) {
           },
           {
             method: 'POST',
+            path: '/updateOrg',
+            handler: function (request, reply) {
+              var id = request.query.id;
```

Comment:
I'm not entirely sure if we should pass the id as a query parameter or in the payload. My first thought would be to add it in the payload which means that we could have only the orgObj pass as an argument (with the id inside the object). Might be worth to see what is the best practice